app/assets/javascripts/developers.js

```diff
@@ -44,7 +44,7 @@ $(function(undefined) {
   (function() {
     var DEVELOPER_TABLE = 'developer-table';
     var dataTables = [
-      DEVELOPER_TABLE
+      DEVELOPER_TABLE, "applications-table"
```

Comment:
we set the visual equivalent of a const for the developer table but not application table. we should be consistant, either: 
```
var DEVELOPER_TABLE = 'developer-table';
var APPLICATIONS_TABLE = 'applications-table';
```

or

```
var dataTables = ['developer-table', 'applications-table'];
```